pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -690,17 +691,68 @@ class StateVectorKokkos final
      * @param opName Name of gate to apply.
      * @param wires Wires to apply gate to.
      * @param inverse Indicates whether to use adjoint of gate.
-     * @param params Optional parameter list for parametric gates.
      */
     auto applyGenerator(const std::string &opName,
                         const std::vector<std::size_t> &wires,
-                        bool inverse = false,
-                        const std::vector<fp_t> &params = {}) -> fp_t {
+                        bool inverse = false) -> fp_t {
```

Comment:
Can we use  `PrecisionT` instead of `fp_t` to ensure the alignment across sv backends?

Reply:
Good catch, updated to match the base class method now.